### ISSUE TITLE
new hint: Amount of mines limits your choices

### ISSUE THREAD

**Issue by havarc** [pwmarcz/kaboom#34]
Would really help to know if the solver already takes the amount of remaining mines into account.

**Comment by havarc** [comment]
![Screenshot from 2023-03-22 00-53-12](https://user-images.githubusercontent.com/28500400/226767562-4a55c8cc-b0cc-4761-a20d-d3eba9cbd40e.png)

**Comment by dieser-niko** [comment]
Yep it does, since the counter is included in pretty much every minesweeper version that I have played.
I had to use the counter a dozen times, not only in kaboom.
